Resources/Descriptives/qml/Descriptives.qml

```diff
@@ -134,8 +134,21 @@ Form
             {
                                     name: "plotVariables";			label: qsTr("Distribution plots")
                 CheckBox {			name: "distPlotDensity";		label: qsTr("Display density")					}
+				CheckBox {			name: "distPlotRug";		label: qsTr("Display rug marks")					}
+				DropDown {
+                name: "binWidthType"
+                label: qsTr("Bin Width Type")
+                indexDefaultValue: 0
+                values:
+                [	
+					{ label: qsTr("Sturges"),				value: "sturges"	},
+                    { label: qsTr("Scott"),					value: "scott"		},
+                    { label: qsTr("Doane"),					value: "doane"	},
+					{ label: qsTr("Freedman-Diaconis"),		value: "fd"	}
+                ]
+				}
             }
-
+			
```

Comment:
please use tabs for .qml files and not spaces. If you're using QtCreator for this, you can adjust the default settings [here](https://doc.qt.io/qtcreator/creator-indenting-code.html#indenting-qml-files)